firebase-vscode/package.json

```diff
@@ -22,12 +22,12 @@
       "properties": {
         "firebase.debug": {
           "type": "boolean",
-          "default": false,
+          "default": true,
```

Comment:
Revert all the changes in this file except the version update in line 6 - all the rest is from running `swap-pkg.js monospace`. We should put a check in at some point to prevent this from being checked in.

Reply:
Done.